packages/opentelemetry-test-utils/package.json

```diff
@@ -33,7 +33,7 @@
     "typescript": "4.3.2"
   },
   "dependencies": {
-    "@opentelemetry/core": "^0.20.0",
-    "@opentelemetry/tracing": "^0.20.0"
+    "@opentelemetry/core": "0.20.0",
```

Comment:
This seems strange. We should not pin core dependencies

Reply:
Agree, i'll close this to avoid merging it. Other PRs didn't seem to have issues :/